app/themes/user-admin/src/app/user-event-log/user-event-log.component.html

```diff
@@ -0,0 +1,58 @@
+<div class="row">
+    <div class="col-xs-12">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <span>User Event Log</span>
+            </div>
+            <div class="panel-body">
+
+                <table class="table table-bordered table-hover">
+                    <thead>
+
+                        <tr>
+                            <th (click)="sortID()">
+                                ID
+                                <i class="fa" [ngClass]="{'fa-sort-asc': (sortDir == 'asc'), 'fa-sort-desc': (sortDir == 'desc')}" aria-hidden="true"></i>
+                            </th>
+                            <th>
+                                Event Log
+                            </th>
+                            <th>
+                                Event Data
+                            </th>
+                            <th>
+                                Created At
+                            </th>
+                        </tr>
+                    </thead>
+
+                    <tbody *ngIf="wait"> 
+                        <tr>
+                            <td colspan="7" align="center">
+                                <i class="fa fa-refresh fa-spin fa-2x fa-fw"></i> 
+                            </td>
+                        </tr>
+                    </tbody>
+
+                    <tbody *ngIf="!wait && events.length <= 0">
+                        <tr>
+                            <td colspan="7" align="center">
+                                <alert type="warning">Sorry, no user events were found. Please try again.</alert>
+                            </td>
+                        </tr>
+                    </tbody>
+
+                    <tbody *ngIf="!wait && events.length > 0">
+                        <tr *ngFor="let event of events">
+                            <td> {{event.id}} </td>
+                            <td> {{event.event}} </td>
+                            <td> {{event.event_data}} </td>
```

Comment:
![useradmin_and_3__node_and_user-admin_ _working_copy__kkennethlee-develop_ _2_changed_files__and_2__mywu__bash_](https://cloud.githubusercontent.com/assets/142324/23563068/f4189a80-fff9-11e6-9f69-8f9b726e9f84.png)

Please render the event_data objects (if its an object) via json. Here is how you can accomplish this:

`{{ event.event_data | json }}`